### DISCUSSION TITLE
safe-settings app deployment

### DISCUSSION THREAD

**Discussion by Kalithasan-Periyasamy** [github/safe-settings#564] [Q&A]
I am seeing multiple deployment options in this link [https://github.com/github/safe-settings/blob/main-enterprise/docs/deploy.md](https://github.com/github/safe-settings/blob/main-enterprise/docs/deploy.md) . 

Is there any way to deploy safe-settings with in github itself ? or can we use safe-settings with out deploy the app in our instance ?

**Comment by svg153** [comment]
To deploy in GitHub, I think it's not possible, read this comment: https://github.com/github/safe-settings/discussions/569#discussioncomment-7871799
To deploy safe-setting is better to do it in:
- vm, check https://github.com/svg153-org/admin/tree/main/apps
- k8s, check https://github.com/github/safe-settings/blob/main-enterprise/docs/deploy.md#deploy-the-app-in-kubernetes

**Reply by Kalithasan-Periyasamy** [reply]
Thanks @svg153
